cranelift/codegen/src/machinst/abi.rs

```diff
@@ -1619,7 +1621,8 @@ impl<M: ABIMachineSpec> Callee<M> {
         sigs: &SigSet,
         vregs: &mut VRegAllocator<M::I>,
     ) -> Option<M::I> {
-        if let Some(i) = sigs[self.sig].stack_ret_arg {
+        let sig_data = &sigs[self.sig];
+        if let Some(i) = sig_data.stack_ret_arg {
```

Comment:
This is a small thing, but could you put this back the way it was before?

```suggestion
        if let Some(i) = sigs[self.sig].stack_ret_arg {
```